concepticondata/conceptlists/Gravina-2014-1611.tsv

```diff
@@ -0,0 +1,1612 @@
+Gravina-2014-1611-693	693	gift	gift	cadeau // cadeau (Cf.: |fv{masalaka}) // cadeau, don // don	2982	GIFT
+Gravina-2014-1611-694	694	gill	gill	ouïes	1916	GILL
+Gravina-2014-1611-695	695	giraffe	giraffe	girafe	3089	GIRAFFE
+Gravina-2014-1611-696	696	girl	girl	(petite) fille, fillette // (petite) fille, fillette,vierge // fille	1646	GIRL
+Gravina-2014-1611-697	697	give	give	donner // tiens! prends!	1447	GIVE
+Gravina-2014-1611-698	698	give pain, hurt	give pain, hurt	faire mal	2411	HURT (SOMEBODY)
+Gravina-2014-1611-699	699	gizzard	gizzard	gésier		
+Gravina-2014-1611-700	700	gnaw	gnaw	ronger // ronger; manger	2706	GNAW
+Gravina-2014-1611-701	701	go	go	aller // aller (beaucoup) // aller, marcher, partir // déplacer; marcher; partir // se mettre en route	695	GO
+Gravina-2014-1611-702	702	go back, return	go back, return	retourner	581	COME BACK
+Gravina-2014-1611-703	703	go round	go round	contourner		
+Gravina-2014-1611-704	704	goat	goat	bouc // chèvre // nombreux en parlant de troupeau	1502	GOAT
+Gravina-2014-1611-705	705	goiter	goiter	goitre	1957	GOITER
+Gravina-2014-1611-706	706	gold	gold	or	1369	GOLD
+Gravina-2014-1611-707	707	good	good	bien // bon // bon; bien // être beau	1035	GOOD
+Gravina-2014-1611-708	708	granary	granary	grenier // grénier	3657	GRANARY
+Gravina-2014-1611-709	709	granddaughter	granddaughter	petite-fille	1619	GRANDDAUGHTER
+Gravina-2014-1611-710	710	grandfather	grandfather	grand-père	1383	GRANDFATHER
+Gravina-2014-1611-711	711	grandmother	grandmother	grand-mère // grandmère	1496	GRANDMOTHER
+Gravina-2014-1611-712	712	grandson	grandson	petit fils // petit-fils	1618	GRANDSON
+Gravina-2014-1611-713	713	grass	grass	(herbe) // fourrage; fourrage ; herbes coupées pour les animaux domestiques // herbe // herbe ; fourage	606	GRASS
+Gravina-2014-1611-714	714	grasshopper	grasshopper	criquet // criquet migrateur // sauterelle	607	GRASSHOPPER
+Gravina-2014-1611-715	715	grave	 // grave	grave // tombe // tombe, tombeau // tombeau // trou pour cuire des pots (deCol); tombe	1773	GRAVE
+Gravina-2014-1611-716	716	gravel	gravel	gravier	3028	GRAVEL
+Gravina-2014-1611-717	717	greet	greet	accueillir // saluer	3552	GREET
+Gravina-2014-1611-718	718	grind	grind	broyer // grincer // moudre // écraser	1033	GRIND
+Gravina-2014-1611-719	719	grinding stone	grinding stone	pierre  moudre // pierre à moudre // pierre à moudre**	2459	GRINDSTONE
+Gravina-2014-1611-720	720	groan (with pain)	groan (with pain)	gémir (de douleur)	716	GROAN
+Gravina-2014-1611-721	721	groom	groom	marié	3282	GROOM
+Gravina-2014-1611-722	722	ground, land	ground, land	sol // terre	2159	GROUND
+Gravina-2014-1611-723	723	groundnut	groundnut	arachide	595	GROUNDNUT (PEANUT)
+Gravina-2014-1611-724	724	grow (of plants)	grow (of plants)	pousser (plantes)	479	GROW
+Gravina-2014-1611-725	725	grow up	grow up	grandir // grandir; grossir	552	GROW UP
+Gravina-2014-1611-726	726	growl	growl	grognement // grogner // gronder	3679	GROWL
+Gravina-2014-1611-727	727	guava	guava	goyave // goyave; goyavier		
```

Comment:
There is a new concept: 3981	GUAVA
So this can be mapped, even if it doesnt appear in the concepticon app yet